src/vivarium_census_prl_synth_pop/data/generate_business_names.py

```diff
@@ -0,0 +1,148 @@
+import sys
+from collections import defaultdict
+
+import numpy as np
+import pandas as pd
+from loguru import logger
+
+from vivarium_census_prl_synth_pop.constants import data_values, paths
+
+
+def generate_business_names_data(n_total_names: str):
+    # loads csv of business names and generates pandas series with random business names
+
+    business_names = pd.read_csv(paths.BUSINESS_NAMES_DATA)
+    bigrams = make_bigrams(business_names)
+
+    # Get frequency of business names and find uncommon ones
+    s_name_freq = business_names.location_name.value_counts()
```

Comment:
could you rename this to be a little clearer? i cant tell what `s` refers to

Reply:
Changed.